dcp_prototype/backend/artifact_manager/schemas/v0.0.0.json

```diff
@@ -0,0 +1,408 @@
+                     "type":"string",
+                     "examples":[
+                        "nervous",
+                        "respiratory"
+                     ]
+                  }
+               },
+               "id":{
+                  "$id":"#/properties/projects/items/properties/id",
+                  "type":"string",
+                  "examples":[
+                     "HCA-Project-cddab57b-6868-4be4-806f-395ed9dd635a"
+                  ]
+               },
+               "title":{
+                  "$id":"#/properties/projects/items/properties/title",
+                  "type":"string",
+                  "examples":[
+                     "Single cell transcriptome analysis of human pancreas"
+                  ]
+               },
+               "library_construction_methods":{
+                  "$id":"#/properties/projects/items/properties/library_construction_methods",
+                  "type":"array",
+                  "items":{
+                     "$id":"#/properties/projects/items/properties/library_construction_methods/items",
+                     "type":"string",
+                     "examples":[
+                        "SS2",
+                        "10x 3' v2"
+                     ]
+                  }
+               },
+               "assay_categories":{
+                  "$id":"#/properties/projects/items/properties/assay_categories",
+                  "type":"array",
+                  "items":{
+                     "$id":"#/properties/projects/items/properties/assay_categories/items",
+                     "type":"string",
+                     "examples":[
+                        "scRNA-seq",
+                        "bulk DNA sequencing"
+                     ]
+                  }
+               },
+               "donor_diseases":{
+                  "$id":"#/properties/projects/items/properties/donor_diseases",
+                  "type":"array",
+                  "items":{
+                     "$id":"#/properties/projects/items/properties/donor_diseases/items",
+                     "type":"string",
+                     "examples":[
+                        "Diseased",
+                        "Normal"
+                     ]
+                  }
+               },
+               "biosample_diseases":{
+                  "$id":"#/properties/projects/items/properties/biosample_diseases",
+                  "type":"array",
+                  "items":{
+                     "$id":"#/properties/projects/items/properties/biosample_diseases/items",
+                     "type":"string",
+                     "examples":[
+                        "Diseased",
+                        "Normal"
+                     ]
+                  }
+               },
+               "selected_cell_types":{
+                  "$id":"#/properties/projects/items/properties/selected_cell_types",
+                  "type":"array",
```

Comment:
Remove the comma at the end of the line.
There are a few other places that have this mistake which you can find by running the scheme through a json reader.

Reply:
Fixed 👍